### DISCUSSION TITLE
How can I install it on my server ?

### DISCUSSION THREAD

**Discussion by crazzysongs** [plausible/analytics#4849] [Self-Hosted Support]
Hi everyone,
How can I install it on my server? I tried all the tutorials from Google, but I failed. After installation, I only see the Nginx default page and can't find the Plausible page.

**Comment by ruslandoga** [comment]
👋 @crazzysongs 

Have you tried the official quick-start guide on https://github.com/plausible/community-edition?

**Comment by crazzysongs** [comment]
brother how can I set base url :

ssh output 


root@vmi2246492:~/plausible-ce# docker compose up -d
[+] Running 31/3
 ✔ plausible_events_db Pulled                                             32.9s
 ✔ plausible Pulled                                                       24.9s
 ✔ plausible_db Pulled                                                    17.0s
[+] Running 8/8
 ✔ Network plausible-ce_default                  Created                   0.1s
 ✔ Volume "plausible-ce_event-data"              Created                   0.0s
 ✔ Volume "plausible-ce_event-logs"              Created                   0.0s
 ✔ Volume "plausible-ce_plausible-data"          Created                   0.0s
 ✔ Volume "plausible-ce_db-data"                 Created                   0.0s
 ✔ Container plausible-ce-plausible_events_db-1  Healthy                   6.2s
 ✔ Container plausible-ce-plausible_db-1         Healthy                   6.2s
 ✔ Container plausible-ce-plausible-1            Started                   6.4s
root@vmi2246492:~/plausible-ce# nano sudo nano plausible-conf.en
root@vmi2246492:~/plausible-ce# BASE_URL=https://plausible.example.com
root@vmi2246492:~/plausible-ce# BASE_URL=http://php.playcraft.live
root@vmi2246492:~/plausible-ce# cat .env
BASE_URL=https://plausible.example.com
SECRET_KEY_BASE=FRc6+6AuzzNko/MAl+U2qooxRt44ntxj4lzRzynl/qF8yHZDeWPL1fYZvKOqhKpZ
HTTP_PORT=80
HTTPS_PORT=443
HTTPS_PORT=443
root@vmi2246492:~/plausible-ce#

**Reply by ruslandoga** [reply]
You can edit it in the `.env` file.

```console
$ nano .env
```

Ask ChatGPT for details :)

**Reply by ruslandoga** [reply]
Note that you would also need to regenerate your `SECRET_KEY_BASE`
